### PR TITLE
Fix link in `numbers_RKI` (Closes #886)

### LINKED ISSUE #886: FAQ: Link about "How often has the Corona warning app been downloaded so far" is not working, pdf file not on rki page
## Where to find the issue
in the file src/data/faq_de.json
in the section 
`                    {
                        "title": "Wie oft wurde die Corona-Warn-App bisher heruntergeladen? Wie viele TeleTANs wurden ausgegeben?",
                        "anchor": "numbers_RKI",
                        "active": true,
                        "textblock": [
                            "Diese und weitere aktuelle Zahlen stellt das Robert Koch-Institut <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Kennzahlen.pdf?__blob=publicationFile' target='_blank' rel='noopener noreferrer'> auf seiner Webseite</a> zur Verfügung."
                        ]
                    },
`
## Describe the issue
The link `<a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Kennzahlen.pdf?__blob=publicationFile' target='_blank' rel='noopener noreferrer'>` didn't work. The pdf file is not on the rki page.

## Suggested change

Talk with the RKI, i didn't find the information on there webpage.

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR corrects a link to the RKI in the FAQ article 
https://www.coronawarn.app/de/faq/#numbers_RKI & 
https://www.coronawarn.app/en/faq/#numbers_RKI

This PR also corrects what has been previously [discussed](https://github.com/corona-warn-app/cwa-website/pull/885#issuecomment-778515262) with @dsarkar in #885.

---

Closes #886

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -801,7 +801,7 @@
                         "anchor": "numbers_RKI",
                         "active": true,
                         "textblock": [
-                            "The Robert Koch Institute provides these and other current numbers <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Kennzahlen.pdf?__blob=publicationFile' target='_blank' rel='noopener noreferrer'>on their website</a> (in German)."
+                            "The Robert Koch Institute provides these and other current numbers <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Archiv_Kennzahlen/WarnApp_KennzahlenTab.html;jsessionid=041B1E316F2E11A2FB9036CD0E457343.internet082?nn=13490888' target='_blank' rel='noopener noreferrer'>on their website</a> (in German)."
```
Comment (heinezen) [inline src/data/faq.json:804]: `jsessionid=041B1E316F2E11A2FB9036CD0E457343.internet082?nn=13490888`

Session ID and nonce do not belong into this link

**Review by heinezen — CHANGES_REQUESTED** [review]
``

File: src/data/faq.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/faq.json:804]: Thanks, only `https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Archiv_Kennzahlen/WarnApp_KennzahlenTab.html`

**Comment by Ein-Tim** [comment]
@heinezen Corrected.

**Review by heinezen — APPROVED** [review]

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim thanks!

**Pull request merged**